lib/routes/airlevel/index.ts

```diff
@@ -0,0 +1,54 @@
+import { Route } from '@/types';
```

Comment:
Please [use namespace](https://docs.rsshub.app/joinus/new-rss/start-code#creating-namespace) `air-level` instead of `airlevel`